.mega-linter.yml

```diff
@@ -0,0 +1,29 @@
+# Configuration file for MegaLinter
+#
+# See all available variables at https://megalinter.io/latest/config-file/ and in
+# linters documentation
+
+# all, none, or list of linter keys
+APPLY_FIXES: all
+
+# If you use ENABLE variable, all other languages/formats/tooling-formats will
+# be disabled by default
+# ENABLE:
+
+# If you use ENABLE_LINTERS variable, all other linters will be disabled by
+# default
+ENABLE_LINTERS:
+  - CPP_CLANG_FORMAT
+
+DISABLE:
+  - COPYPASTE # Comment to enable checks of excessive copy-pastes
```

Comment:
Why is this disabled?

Reply:
Only clang-format is enabled, everything else is disabled (for now). That explicit setting is disabled from the install wizard.  We can revisit later.